src/plasmapy/diagnostics/charged_particle_radiography/synthetic_radiography.py

```diff
@@ -87,7 +87,7 @@ def __init__(
             "detector": (u.m, "attribute"),
             "mag": (u.m, "attribute"),
             "max_deflection": (None, "attribute"),
-            "nparticles": (None, "attribute"),
+            "num_particles": (None, "attribute"),
```

Comment:
Would this count as a breaking change?

Reply:
Maybe? It's technically user-facing, although I at least never access it. I'll add a changelog for it.